mandatory/2-function-creation.js

```diff
@@ -25,7 +29,9 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  return arrayOfStrings.map((x) => x.trim().toLowerCase().replace(/\//g, ""));
```

Comment:
Try to give variables meaningful names e.g. in this case `x` could be `string`. Otherwise great. 

You used regex here (`/\//g`) which is perfectly fine. You could have just have a straightforward string if you'd preferred: `.replace('/', '')`

Reply:
(^ but that would only replace one /)